feature-libs/quote/styles/_quote-links.scss

```diff
@@ -24,6 +24,14 @@
             border: none;
             padding-inline-end: 0;
           }
+
+          button {
```

Comment:
@suprishi, please, have a look at this styling changes.
Only the color should be changed for a button to black.

Reply:
Looks good to me, thanks